src/library/searchquery.h

```diff
@@ -57,6 +57,7 @@ class AndNode : public GroupNode {
 class NotNode : public QueryNode {
   public:
     explicit NotNode(QueryNode* pNode);
+    ~NotNode();
```

Comment:
The refactoring for master will render all destructors of derived classes obsolete, btw ;)